### PR TITLE
UI: Stitching with `LabeledBadge`

### PR DESCRIPTION
**Pull request by kpyszkowski**
Ref: #36 

This pull request introduces the following changes:
- [Updated `LabeledBadge` font size](https://github.com/threshold-network/bitcoin-on-base/commit/0006d08fd5855a81350973a48e46081b41945d70)
- [Added elapsed time badge in `MintingDepositTimeline`](https://github.com/threshold-network/bitcoin-on-base/commit/d12e7eb7b56bcb9fd7e452f006c9bff153a45335) 
- [Used `LabeledBadge` component to follow designs](https://github.com/threshold-network/bitcoin-on-base/commit/2ca4d93a0cac8a97197038322ccb0b3bbd36344e)

### CODE REVIEW

File: src/pages/tBTC/Bridge/Minting/MintingDepositTimeline.tsx [hunk 1]
```diff
@@ -190,11 +192,16 @@ export const MintingDepositTimeline: FC<MintingTimelineProps> = ({
...
         fontWeight="medium"
-        mb={isOnDepositDetailsPage || account ? 10 : 6}
+        mb={isOnDepositDetailsPage ? 2 : account ? 10 : 6}
         lineHeight={6}
       >
         {title}
       </BodyLg>
+      {isOnDepositDetailsPage && (
+        <LabeledBadge icon={ClockIcon} label="Elapsed time" mb={10}>
+          3 hrs
```
Comment (michalsmiarowski) [inline src/pages/tBTC/Bridge/Minting/MintingDepositTimeline.tsx:202]: Non-blocking but good to have:

We should add a TODO here. This value should be calculated based on the time where the reveal was done (but in a separate PR). I think we've done a similar thing in token-dashbard for redemption. 

We can also create a task for it.

**Review by michalsmiarowski — APPROVED** [review]
LGTM

Left one non-blocking comments

File: src/pages/tBTC/Bridge/Minting/MintingDepositTimeline.tsx (hunk 1 above)
Comment (kpyszkowski) [reply src/pages/tBTC/Bridge/Minting/MintingDepositTimeline.tsx:202]: Are you talking about this [Badge](https://github.com/threshold-network/token-dashboard/blob/265f38948840e745171b6368e28942f96be8197e/src/pages/tBTC/Bridge/DepositDetails.tsx#L252C1-L262C25)?
It's hardcoded in token-dashboard

Comment (kpyszkowski) [reply src/pages/tBTC/Bridge/Minting/MintingDepositTimeline.tsx:203]: Added TODO comment: 
Ref commit: bb0d020fbedf3638cb5ee907d16c18562a6ed7c3

**Review by michalsmiarowski — APPROVED** [review]
LGTM 🔥

**Pull request merged**